python/mxnet/recordio.py

```diff
@@ -17,14 +17,19 @@
     cv2 = None
 
 class MXRecordIO(object):
-    """Read/write RecordIO format data.
+    """Reads/writes `RecordIO` data format, supporting sequential read and write.
+
+    Example usage:
+    ----------
+    >>> record = mx.recordio.MXRecordIO('tmp.rec', 'w')
```

Comment:
read/write example is added in the respective function. If I add it here, it might be redundant? I added whole example now so that one should able to run it.